### ISSUE TITLE
[CAIA A11Y Research]: Ask VA Va.gov Team, Ask VA Screen Magnification - (First Study April 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#82856]
> [!NOTE]
> ## First Round of AT Research for Ask VA**
> - **Initial Kickoff**: `Date and Time TBD`
> - **Research Timeframe**: Tuesday, April 23 - Friday, April 26
> - **Time of Day:** `TBD`
> - **Initial Pilot**:  `4/30/2024 in Figma` #81692
> - **Artifacts Available**: `See Supporting Artifacts` #80535
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)
@coforma-jamie 
@sara-amanda 

## About the ASK VA Team
- Team name: Ask VA VA.gov
- OCTO-DE product owner: Becky Phung
- Product manager: Holly Hansen
- Designer: Tyler Gindraux, Thomas Michaud
- FE engineer: Hemesh Patel, Eddie Otero
- Product/team Slack channel: #ask-va-public

## Next Steps

After discussing the team's needs with Tyler, we determined the following:

### Completed
- [x] Tyler will run a screen magnification pilot with a CAIA a11y using the team's Figma prototypes. 
- [x] The team will run its study without AT users, and without CAIA support.

### Later

> [!CAUTION]
> While we’re waiting on everything to be ready in staging, we’ll probably sneak in one more research study using Figma. I’m guessing we’re looking at July for next steps testing with screen mag and screen reader users.
>yes! we have a ticket for a SR pilot for sure.

- The team will create a coded prototype in Codepen or Staging.
- Tyler will run two screen reader pilots with CAIA a11ys - one on mobile, one on desktop.
- The team will run a new study with AT users - magnification and screen reader - with CAIA support.
- Refer to EPIC #80535

**Comment by sara-amanda** [comment]
## Research Findings Update 

**DSVA Slack Thread** 🧵 [Findings from April 2024 were presented in the 7/8/2024 Design + Research + Content Sync meeting.](https://dsva.slack.com/archives/C0NGDDXME/p1720443623691809)

### Presentation Slides

[PDF of Slides Presented from the April 2024 Research](https://dsva.slack.com/files/U04775XBW8H/F07BGTQ3VBL/monday_design_and_research_share_-_ask_va_-_july_8.pdf)

### Experimental Design

> **Tyler Gindraux:** Btw, here's our [experimental design ticket for the progress bar](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2820). In case this provides anymore detail. Next step is testing with SR users in staging, and we'll test with the % complete heading being the first element in focus on the page.

## Assistive Technology
Second study will include AT, and is scheduled to take place in August 2024. See ticket: #87480 

cc: @coforma-jamie @SarahKay8 @EvanAtCoforma
